ci/params/aurora/quickstart-jira-dc-aurora-params.json

```diff
@@ -33,6 +33,6 @@
   },
   {
     "ParameterKey": "DBInstanceClass",
-    "ParameterValue": "db.r4.large"
+    "ParameterValue": "db.r5.large"
```

Comment:
I just doublechecked that the `9.6.12` version of Aurora supports `r5` family (the previous version didn't).

https://github.com/aws-quickstart/quickstart-atlassian-services/blob/c64cea657b2f55c0a96a12f58b867a34c1b724df/templates/quickstart-database-for-atlassian-services.yaml#L153